### DISCUSSION TITLE
7900 GRE has ROCm support but not HIP SDK

### DISCUSSION THREAD

**Discussion by utorque** [ROCm/ROCm#3316] [Q&A] [answered]
Hello,  
I just wanted to confirm that the official infos are correct. Is the 7900 GRE supported by ROCm on linux but NOT supported on windows for HIP SDK ? That seems surprising. Some links :  
[Win compatibility matrix](https://rocm.docs.amd.com/projects/install-on-windows/en/latest/reference/system-requirements.html)  
[Linux compatibility matrix](https://rocm.docs.amd.com/projects/install-on-linux/en/latest/reference/system-requirements.html)  
Can anyone confirm ? If so, is there some insight on why it is not compatible with HIP SDK ?

**Comment by tomashomecat** [comment]
Hi
i havent 7900gre so cannot check, but am confident for 99.99% that main parameter is architecture code, (in this case gfx1100) and commercial names are given in those tables just for information. So on Your place i would buy this very good card and will sell on ebay if despite expectations it will be incompatible.

**Comment by darren-amd** [comment, marked as answer]
Hi @Utorque,

Thanks for bringing this up, since the 7900 GRE is based on a similar architecture to the other cards (like the 7900 XTX), it might work with the HIP SDK. I'm currently contacting the internal team to confirm compatibility and perhaps updating the documentation if it is indeed compatible. In the meantime, please feel free to give it a try and let me know if you run into any issues, thanks!

EDIT: Spoke with the internal team and the 7900 GRE is not officially supported for the HIP SDK on Windows, as we do not currently test on it. As above, it should work in theory, so please give it a try and let me know, thanks!

**Reply by haipz** [reply]
I installed hip sdk but seems it doesn't support 7900gre.

**Reply by darren-amd** [reply]
Hi @haipz, did you follow the instructions available [here](https://rocm.docs.amd.com/projects/install-on-windows/en/latest/)? What issue are you running into?
